File: tirefitting/modules/core/src/com/company/tirefitting/core/RubberFix.java
package com.company.tirefitting.core;

import org.springframework.stereotype.Component;

@Component(RubberFix.NAME)
public class RubberFix implements RequestTypeDurationCalculator {
    public static final String NAME = "tirefitting_RubberFix";

    @Override
    public double getDuration(int radius) {
        return 30 * 60 * (1 + ((double)radius - 13) / 5);
    }
}

File: tire_fitting/lib/requestTypes/RubberFix.dart
import 'package:tire_fitting/requestTypes/RequestType.dart';

class RubberFix extends RequestType{
  @override
  double getDuration(int radius) {
    return 30 * 60 * (1 + (radius - 13) / 5);
  }

  String name = 'RUBBER_FIX';

}